### PR TITLE
add contributors and lurkers to readme

### PR DESCRIPTION
**Pull request by Gakkenyak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,3 +127,12 @@ The best way to learn Redwood is by going through the comprehensive [tutorial](h
 
 - Stay updated: read [Forum announcements](https://community.redwoodjs.com/c/announcements/5), follow us on [Twitter](https://twitter.com/redwoodjs), and subscribe to the [newsletter](https://redwoodjs.com/newsletter)
 - [Learn how to contribute](https://redwoodjs.com/docs/contributing)
+
+## Contributors
+- Ainab
```
Comment (drikusroor) [inline README.md:132]: It's common practice in markdown to leave a whitespace between a heading and the underlying text

**Review by drikusroor — APPROVED** [review]
LGTM, I'll leave it up to you if you want to improve it or not

**Review by naomi-eliasar — APPROVED** [review]

**Comment by drikusroor** [comment]
Congratulations on your first PR on Github! 🚀  Here's a cat gif to celebrate:

**Review by adrianabm — APPROVED** [review]
LGTM :shipit:

**Pull request merged**